apps/hash-graph/libs/graph/src/store/fetcher.rs

```diff
@@ -1237,44 +1238,20 @@ where
         .await?;
 
         self.store
-            .create_entity(
-                actor_id,
-                authorization_api,
-                temporal_client,
-                owned_by_id,
-                entity_uuid,
-                decision_time,
-                archived,
-                draft,
-                entity_type_id,
-                properties,
-                link_data,
-                relationships,
-            )
+            .create_entity(actor_id, authorization_api, temporal_client, params)
             .await
     }
 
-    #[tracing::instrument(level = "info", skip(self, authorization_api))]
+    #[tracing::instrument(level = "info", skip(self, authorization_api, params))]
```

Comment:
Why? I feel like we're omitting useful additional information here.

Reply:
Actually, I removed this instrumentation as this is not really relevant. It's just calling the underlying function which has an instrumentation as well. I removed `params` there.